lib/archethic/crypto.ex

```diff
@@ -1175,13 +1175,31 @@ defmodule Archethic.Crypto do
     end
   end
 
+  def get_root_ca_public_key(<<0::8, _::8, _::binary>>) do
```

Comment:
I think you can use general case with 
```suggestion
  def get_root_ca_public_key(_), do: ""
```